com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -180,25 +203,35 @@ internal void __endSendClientRpc(NetworkSerializer serializer, uint rpcMethodId,
                 return;
             }
 
-            var rpcQueueContainer = NetworkManager.RpcQueueContainer;
+            if (clientRpcParams.Send.UpdateStage == NetworkUpdateStage.Unset)
+            {
+                clientRpcParams.Send.UpdateStage = NetworkUpdateLoop.UpdateStage;
+
+                if (clientRpcParams.Send.UpdateStage == NetworkUpdateStage.Initialization)
+                {
+                    clientRpcParams.Send.UpdateStage = NetworkUpdateStage.EarlyUpdate;
+                }
+            }
```

Comment:
Another repeating region (this one is for `clientRpcParams`) that needs a function.

Reply:
Done.